pages/common/sass.md

```diff
@@ -0,0 +1,12 @@
+# Sass
+
+> Converts SCSS or Sass files to CSS
+> .scss and .sass can be used interchangeably in below examples depending on your preference
+
+- Immediately one time convert SCSS or Sass file to CSS
+
+`sass {{inputfile}}.scss:{{outputfile}}.css`
```

Comment:
As far as I understood from documentation, there should be space, not a colon:

`sass [options] [INPUT] [OUTPUT]`